MAINTAINERS.md

```diff
@@ -23,13 +23,20 @@ This is a list of maintainers for this project. See [CODEOWNERS](/.github/CODEOW
 * Brett Rosenblatt
 * Brandon Tabaska
 * Ebuwa Evbuoma-Fike
+* James Bursa
+* Ryan Lewis
 
 ## Product and Project Management
 
-* Esther Oke
-* Sumit Haiveettil
+* Sumi Thaiveettil
```

Comment:
Checks history to see who made this mistake... https://github.com/HHS/simpler-grants-gov/commit/72db5cf42504d99b7c629b6e5ee5f7e4c65bc862 😭 . Apologies!!!